gnosis/gp-ui
# 92/refactor order widget structure

# Summary

Part of #92 
Depends on #145 (:sweat_drops: :fallen_leaf:)

Moved OrderDetails to `components/orders` and started refactoring it.

New sub-component `OrderPriceDisplay`, with associated storybook stories.
![screenshot_2021-02-12_14-56-56](https://user-images.githubusercontent.com/43217/107831325-acd76b00-6d42-11eb-8d58-2928962eca75.png)


# Testing

- Check the storybook https://pr146--gpui.review.gnosisdev.com/storybook/?path=/docs/orders-orderpricedisplay--default
- Check the live app https://pr146--gpui.review.gnosisdev.com/rinkeby/orders/0xa6223509c0a41f992995d8781c3452ae8e2f55640b21d057fc02c7fb5a1674b35b0abe214ab7875562adee331deff0fe1912fe4260246928/

Switch button fully self-contained.
Inverting the price does not cause OrderDetails re-render.
